### PR TITLE
taylor|style(minor-adjustments):

### PR DESCRIPTION
**Pull request by taylorjdawson**
- Removes buy functionality--links to OS now
 - Change default height to 200px
 - Change default orientation to horizontal
 - Add payment token img to previous price
 - Simplify price template logic

### CODE REVIEW

File: src/nft-card-front.ts [hunk 1]
```diff
@@ -202,24 +222,15 @@ export class NftCardFrontTemplate extends LitElement {
...
-      // @ts-ignore ignore until LastSale type gets added to SDK
-      const formattedPrevPrice = this.asset.lastSale.total_price / Math.pow(10, this.asset.lastSale.payment_token.decimals)
-      // @ts-ignore ignore until LastSale type gets added to SDK
-      const prevPriceSymbol = this.asset.lastSale.payment_token.symbol === 'ETH' ? 'Ξ ' : ''
-      prevPriceTemplate = html`<div class="asset-detail-price-previous">Prev. ${prevPriceSymbol} ${formattedPrevPrice}</div>`
-    }
+    const prevPriceTemplate = this.lastSaleData?.paymentToken ?
+        NftCardFrontTemplate.getPriceTemplate(PriceType.Previous, this.lastSaleData.paymentToken,
+              // @ts-ignore ignore
+            this.asset?.lastSale?.total_price / Math.pow(10, this.asset?.lastSale?.payment_token.decimals)) : null
```
Comment (alexanderatallah) [inline src/nft-card-front.ts]: why isn't this `totalPrice`? thought that was being serialized now

Comment (alexanderatallah) [reply src/nft-card-front.ts]: same with `paymentToken`

File: src/nft-card-front.ts [hunk 2]
```diff
@@ -202,24 +222,15 @@ export class NftCardFrontTemplate extends LitElement {
...
+            this.lastSaleData.currentPrice) : null
 
-    if (this.asset?.lastSale) {
-      // @ts-ignore ignore until LastSale type gets added to SDK
-      const formattedPrevPrice = this.asset.lastSale.total_price / Math.pow(10, this.asset.lastSale.payment_token.decimals)
-      // @ts-ignore ignore until LastSale type gets added to SDK
-      const prevPriceSymbol = this.asset.lastSale.payment_token.symbol === 'ETH' ? 'Ξ ' : ''
-      prevPriceTemplate = html`<div class="asset-detail-price-previous">Prev. ${prevPriceSymbol} ${formattedPrevPrice}</div>`
-    }
+    const prevPriceTemplate = this.lastSaleData?.paymentToken ?
```
Comment (alexanderatallah) [inline src/nft-card-front.ts]: why aren't we checking `this.asset?`'s paymentToken and using it? seems like there's two ways to check the lastSale here. is lastSaleData no longer suppsed to be used

File: src/nft-card-front.ts [hunk 3]
```diff
@@ -202,24 +222,15 @@ export class NftCardFrontTemplate extends LitElement {
...
-    let currentPriceTemplate: TemplateResult = html``
-
-    if (this.lastSaleData?.paymentToken) {
-      // const currentPriceSymbol = this.lastSaleData.paymentToken.symbol === 'ETH' ? 'Ξ ' : ''
-      currentPriceTemplate = this.getCurrentPriceTemplate(this.lastSaleData.paymentToken, this.lastSaleData.currentPrice)
 
-    }
+    const currentPriceTemplate = this.lastSaleData?.paymentToken ?
+        NftCardFrontTemplate.getPriceTemplate(PriceType.Current, this.lastSaleData.paymentToken,
+            this.lastSaleData.currentPrice) : null
```
Comment (alexanderatallah) [inline src/nft-card-front.ts]: why is the "currentPrice" under a variable called `lastSaleData`? that's very confusing. we have `this.asset` in this component right? the top asset.sellOrder would have the `currentPrice`

File: package.json [hunk 4]
```diff
@@ -11,18 +11,18 @@
...
-    "url": "https://github.com/ProjectOpenSea/embeddable_nfts/issues"
+    "url": "https://github.com/ProjectOpenSea/embeddable-nfts/issues"
   },
-  "homepage": "https://github.com/ProjectOpenSea/embeddable_nfts#readme",
+  "homepage": "https://github.com/ProjectOpenSea/embeddable-nfts#readme",
   "dependencies": {
     "lit-element": "^2.2.1",
     "lit-html": "^1.1.2",
-    "opensea-js": "^0.7.1",
+    "opensea-js": "^1.0.0-rc6",
```
Comment (alexanderatallah) [inline package.json]: let's update this to ^1.0.0

Comment (taylorjdawson) [reply package.json]: @alexanderatallah updated!

**Comment by alexanderatallah** [comment]
@taylorjdawson sorry one more change: let's use opensea-js v1.1.0, which does extra checking on fees to prevent weird fee usage

**Pull request merged**